helm/alfresco-infrastructure/templates/pv-nfs.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.persistence.efs.enabled }}
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ template "alfresco-infrastructure.fullname" . }}
+provisioner: alfresco.com/pv
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: pv-{{ template "alfresco-infrastructure.fullname" . }}
+spec:
+  capacity:
+    storage: {{ .Values.persistence.baseSize | quote }}
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: {{ .Values.persistence.reclaimPolicy | default "Recycle" }}
+  nfs:
+    server: "{{ .Values.persistence.efs.dns }}"
```

Comment:
I guess it would potentially be a breaking change if we also made these more generic: `persistence.nfs.dns`?

Reply:
Sadly yes. we will fix this with the next release.